runtime/doc/lua.txt

```diff
@@ -3670,4 +3670,61 @@ totable({f}, {...})                                       *vim.iter.totable()*
     Return: ~
         (table)
 
+
+==============================================================================
+Lua module: vim.snippet                                          *vim.snippet*
+
+active()                                                    *snippet.active()*
+    Returns `true` if there's an active snippet in the current buffer.
+
+    Return: ~
+        (boolean)
+
+exit()                                                        *snippet.exit()*
+    Exits the current snippet.
+
+expand({input})                                             *snippet.expand()*
+    Expands the given snippet text. Refer to https://microsoft.github.io/language-server-protocol/specification/#snippet_syntax for the specification of valid input.
+
+    Tabstops are highlighted with hl-SnippetTabstop.
+
+    Parameters: ~
+      • {input}  (string)
+
+jump({direction})                                             *snippet.jump()*
+    Jumps (if possible) within the active snippet in the given direction. You
```

Comment:
maybe mention very briefly: if not possible, false is returned (instead of throwing an error, or something)